scripts/eda_kube.sh

```diff
@@ -22,6 +22,11 @@ source "${SCRIPTS_DIR}"/common/utils.sh
 
 trap handle_errors ERR
 
+export EDA_IMAGE=${EDA_IMAGE:-'quay.io/ansible/eda-server:main'}
+export EDA_UI_IMAGE=${EDA_UI_IMAGE:-'quay.io/ansible/eda-ui:main'}
+export EDA_IMAGE_PULL_POLICY=${EDA_IMAGE_PULL_POLICY:-'Always'}
+export EDA_UI_IMAGE_PULL_POLICY=${EDA_UI_IMAGE_PULL_POLICY:-'Always'}
```

Comment:
Conceivably these could be made to automatically set `Never` if no specific value was set and the associated image starts with `localhost`.

Reply:
@jshimkus-rh when I do build using 
```
task minikube:build:api
```
The image is called 
```
docker.io/library/aap-eda:latest
```

Reply:
Never mind, then.